packages/idos-sdk-js/src/lib/idos.ts

```diff
@@ -240,4 +240,9 @@ export class idOS {
       await this.updateAttributesIfNeeded(filteredUserAttributes, litSavableAttributes);
     });
   }
+
+  async discoverEncryptionKey() {
+    const response = await this.enclave.discoverUserEncryptionKey();
+    return response;
```

Comment:
Ditto be mindful about forced `await`s.
```suggestion
    return this.enclave.discoverUserEncryptionKey();
```